Berksfile

```diff
@@ -12,5 +12,4 @@ if RUBY_VERSION.to_f < 2.0
   cookbook 'ohai', '< 4.0'
 end
 
-cookbook 'hadoop', git: 'https://github.com/caskdata/hadoop_cookbook.git', branch: 'master'
```

Comment:
what's the reason for this?  I assume this means we must test against the release version, and not whatever's merged to master, which means we can't release these cookbooks together.  not sure what a use-case for that would be though.